lms/urls.py

```diff
@@ -820,10 +820,11 @@
 
 if settings.FEATURES.get('ENABLE_OAUTH2_PROVIDER'):
     urlpatterns += (
-        url(r'^oauth2/', include('oauth2_provider.urls', namespace='oauth2')),
+        url(r'^oauth2/', include('lms.djangoapps.oauth_dispatch.urls')),
+        url(r'^oauth2/', include('edx_oauth2_provider.urls', namespace='oauth2')),
```

Comment:
Suggestion: Please add a comment to specify which one is for DOT and which one is for DOP.  Many times we look through the URLs file as our top-level view of the views, and boy is the naming confusing - constricted again by Python.